[2016-12-15 16:32:34] production.ERROR: Symfony\Component\Console\Exception\CommandNotFoundException: Command "reserve" is not defined.

Did you mean this?
    serve in C:\xampp\htdocs\myWeb\private\vendor\symfony\console\Application.php:536
Stack trace:
#0 C:\xampp\htdocs\myWeb\private\vendor\symfony\console\Application.php(184): Symfony\Component\Console\Application->find('reserve')
#1 C:\xampp\htdocs\myWeb\private\vendor\symfony\console\Application.php(118): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#2 C:\xampp\htdocs\myWeb\private\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(122): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#3 C:\xampp\htdocs\myWeb\private\artisan(35): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#4 {main}  
[2016-12-15 16:32:39] production.ERROR: Symfony\Component\Console\Exception\CommandNotFoundException: Command "reserv" is not defined. in C:\xampp\htdocs\myWeb\private\vendor\symfony\console\Application.php:536
Stack trace:
#0 C:\xampp\htdocs\myWeb\private\vendor\symfony\console\Application.php(184): Symfony\Component\Console\Application->find('reserv')
#1 C:\xampp\htdocs\myWeb\private\vendor\symfony\console\Application.php(118): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#2 C:\xampp\htdocs\myWeb\private\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(122): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#3 C:\xampp\htdocs\myWeb\private\artisan(35): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#4 {main}  
[2016-12-15 16:32:45] production.ERROR: Symfony\Component\Console\Exception\CommandNotFoundException: Command "reserve" is not defined.

Did you mean this?
    serve in C:\xampp\htdocs\myWeb\private\vendor\symfony\console\Application.php:536
Stack trace:
#0 C:\xampp\htdocs\myWeb\private\vendor\symfony\console\Application.php(184): Symfony\Component\Console\Application->find('reserve')
#1 C:\xampp\htdocs\myWeb\private\vendor\symfony\console\Application.php(118): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#2 C:\xampp\htdocs\myWeb\private\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(122): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#3 C:\xampp\htdocs\myWeb\private\artisan(35): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#4 {main}  
[2016-12-15 16:33:00] production.ERROR: ErrorException: chdir(): No such file or directory (errno 2) in C:\xampp\htdocs\myWeb\private\vendor\laravel\framework\src\Illuminate\Foundation\Console\ServeCommand.php:35
Stack trace:
#0 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(2, 'chdir(): No suc...', 'C:\\xampp\\htdocs...', 35, Array)
#1 C:\xampp\htdocs\myWeb\private\vendor\laravel\framework\src\Illuminate\Foundation\Console\ServeCommand.php(35): chdir('C:\\xampp\\htdocs...')
#2 [internal function]: Illuminate\Foundation\Console\ServeCommand->fire()
#3 C:\xampp\htdocs\myWeb\private\bootstrap\cache\compiled.php(1272): call_user_func_array(Array, Array)
#4 C:\xampp\htdocs\myWeb\private\vendor\laravel\framework\src\Illuminate\Console\Command.php(169): Illuminate\Container\Container->call(Array)
#5 C:\xampp\htdocs\myWeb\private\vendor\symfony\console\Command\Command.php(256): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#6 C:\xampp\htdocs\myWeb\private\vendor\laravel\framework\src\Illuminate\Console\Command.php(155): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#7 C:\xampp\htdocs\myWeb\private\vendor\symfony\console\Application.php(820): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#8 C:\xampp\htdocs\myWeb\private\vendor\symfony\console\Application.php(187): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Foundation\Console\ServeCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#9 C:\xampp\htdocs\myWeb\private\vendor\symfony\console\Application.php(118): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#10 C:\xampp\htdocs\myWeb\private\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(122): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#11 C:\xampp\htdocs\myWeb\private\artisan(35): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#12 {main}  
[2016-12-15 16:37:30] production.ERROR: ErrorException: chdir(): No such file or directory (errno 2) in C:\xampp\htdocs\myWeb\private\vendor\laravel\framework\src\Illuminate\Foundation\Console\ServeCommand.php:35
Stack trace:
#0 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(2, 'chdir(): No suc...', 'C:\\xampp\\htdocs...', 35, Array)
#1 C:\xampp\htdocs\myWeb\private\vendor\laravel\framework\src\Illuminate\Foundation\Console\ServeCommand.php(35): chdir('C:\\xampp\\htdocs...')
#2 [internal function]: Illuminate\Foundation\Console\ServeCommand->fire()
#3 C:\xampp\htdocs\myWeb\private\bootstrap\cache\compiled.php(1272): call_user_func_array(Array, Array)
#4 C:\xampp\htdocs\myWeb\private\vendor\laravel\framework\src\Illuminate\Console\Command.php(169): Illuminate\Container\Container->call(Array)
#5 C:\xampp\htdocs\myWeb\private\vendor\symfony\console\Command\Command.php(256): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#6 C:\xampp\htdocs\myWeb\private\vendor\laravel\framework\src\Illuminate\Console\Command.php(155): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#7 C:\xampp\htdocs\myWeb\private\vendor\symfony\console\Application.php(820): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#8 C:\xampp\htdocs\myWeb\private\vendor\symfony\console\Application.php(187): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Foundation\Console\ServeCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#9 C:\xampp\htdocs\myWeb\private\vendor\symfony\console\Application.php(118): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#10 C:\xampp\htdocs\myWeb\private\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(122): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#11 C:\xampp\htdocs\myWeb\private\artisan(35): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#12 {main}  
[2016-12-15 16:38:59] production.ERROR: ErrorException: chdir(): No such file or directory (errno 2) in C:\xampp\htdocs\myWeb\private\vendor\laravel\framework\src\Illuminate\Foundation\Console\ServeCommand.php:35
Stack trace:
#0 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(2, 'chdir(): No suc...', 'C:\\xampp\\htdocs...', 35, Array)
#1 C:\xampp\htdocs\myWeb\private\vendor\laravel\framework\src\Illuminate\Foundation\Console\ServeCommand.php(35): chdir('C:\\xampp\\htdocs...')
#2 [internal function]: Illuminate\Foundation\Console\ServeCommand->fire()
#3 C:\xampp\htdocs\myWeb\private\bootstrap\cache\compiled.php(1272): call_user_func_array(Array, Array)
#4 C:\xampp\htdocs\myWeb\private\vendor\laravel\framework\src\Illuminate\Console\Command.php(169): Illuminate\Container\Container->call(Array)
#5 C:\xampp\htdocs\myWeb\private\vendor\symfony\console\Command\Command.php(256): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#6 C:\xampp\htdocs\myWeb\private\vendor\laravel\framework\src\Illuminate\Console\Command.php(155): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#7 C:\xampp\htdocs\myWeb\private\vendor\symfony\console\Application.php(820): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#8 C:\xampp\htdocs\myWeb\private\vendor\symfony\console\Application.php(187): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Foundation\Console\ServeCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#9 C:\xampp\htdocs\myWeb\private\vendor\symfony\console\Application.php(118): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#10 C:\xampp\htdocs\myWeb\private\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(122): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#11 C:\xampp\htdocs\myWeb\private\artisan(35): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#12 {main}  
[2016-12-15 16:40:00] production.ERROR: ErrorException: chdir(): No such file or directory (errno 2) in C:\xampp\htdocs\myWeb\private\vendor\laravel\framework\src\Illuminate\Foundation\Console\ServeCommand.php:35
Stack trace:
#0 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(2, 'chdir(): No suc...', 'C:\\xampp\\htdocs...', 35, Array)
#1 C:\xampp\htdocs\myWeb\private\vendor\laravel\framework\src\Illuminate\Foundation\Console\ServeCommand.php(35): chdir('C:\\xampp\\htdocs...')
#2 [internal function]: Illuminate\Foundation\Console\ServeCommand->fire()
#3 C:\xampp\htdocs\myWeb\private\bootstrap\cache\compiled.php(1272): call_user_func_array(Array, Array)
#4 C:\xampp\htdocs\myWeb\private\vendor\laravel\framework\src\Illuminate\Console\Command.php(169): Illuminate\Container\Container->call(Array)
#5 C:\xampp\htdocs\myWeb\private\vendor\symfony\console\Command\Command.php(256): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#6 C:\xampp\htdocs\myWeb\private\vendor\laravel\framework\src\Illuminate\Console\Command.php(155): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#7 C:\xampp\htdocs\myWeb\private\vendor\symfony\console\Application.php(820): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#8 C:\xampp\htdocs\myWeb\private\vendor\symfony\console\Application.php(187): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Foundation\Console\ServeCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#9 C:\xampp\htdocs\myWeb\private\vendor\symfony\console\Application.php(118): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#10 C:\xampp\htdocs\myWeb\private\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(122): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#11 C:\xampp\htdocs\myWeb\private\artisan(35): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#12 {main}  
